infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/runtime/LogReplicationServerRouter.java

```diff
@@ -99,7 +97,7 @@ public void sendResponse(CorfuMsg inMsg, CorfuMsg outMsg) {
         log.trace("Ready to send response {}", outMsg.getMsgType());
         outMsg.copyBaseFields(inMsg);
         try {
-            serverAdapter.send(CorfuMessageConverterUtils.toProtoBuf(outMsg));
+            //serverAdapter.send(CorfuMessageConverterUtils.toProtoBuf(outMsg));
```

Comment:
should we get rid of this method?

Reply:
This will be purged in the subsequent PR: https://github.com/CorfuDB/CorfuDB/pull/2869